### ISSUE TITLE
The formation of a package with the contents

### ISSUE THREAD

**Issue by ShirokiyAlex** [xargsgrep/PortKnocker#20]
You can add to the program the ability to send a package with content (text), for devices that analyze the content of the package is a big plus in security.

**Comment by TDFKAOlli** [comment]
I think you didn't understand the concept. Packets are not inspected its just filtering for connection attempt to specific ports. There are 65535 ports and from a sequence of 3 ports I think there are 281.462.092.005.375 possible combinations (if I'm getting math correctly here), which is for sure save enough I guess. ;-)

**Comment by ShirokiyAlex** [comment]
There's no such thing as a lot of security, my boss thinks. For Windows, there is a similar program that sends a text package to the desired port.  The router analyzes the packet by Layer7 Protocols if the sequence of ports matches and the contents of the packets allow access.
![image](https://user-images.githubusercontent.com/42232434/45609701-8a917400-ba48-11e8-9a8e-6f5f34ed64de.JPG)

**Comment by TDFKAOlli** [comment]
Yep, I know. :-) Still there might be a point where you think it is reasonable good. ;-)

Anyhow I had a look at the knockd (server) and it looks like it is possible. Packet inspection code seems to also get the tcp packet itself. So I was wrong with my statement above.

**Comment by ShirokiyAlex** [comment]
Будем ждать добавления этой функции, спасибо.

**Comment by ShirokiyAlex** [comment]
We look forward to adding this feature, thank you.
